src/config/locales/en/error.js

```diff
@@ -1137,6 +1137,10 @@ export const error = {
       title: 'Your response is required',
       message: '',
     },
+    duplicate: {
+      title: 'There is a problem with your emails',
+      message: 'Please provide different emails.'
```

Comment:
@michaelccata flagging this as new error message copy, does this need to be approved?

Reply:
Thanks for flagging @suzubara 
LGTM